sdk/python/feast/feature_store.py

```diff
@@ -1876,15 +1876,14 @@ def retrieve_online_documents_v2(
             features: The list of features that should be retrieved from the online document store. These features can be
                 specified either as a list of string document feature references or as a feature service. String feature
                 references must have format "feature_view:feature", e.g, "document_fv:document_embeddings".
-            query: The query to retrieve the closest document features for.
+            query: The query to retrieve the closest document features for (optional)
```

Comment:
Probably it's `The embedded query` rather than `The query`

Reply:
Also missed to doc the `query_string` param

Reply:
Good catch!